src/components/forms/inputs/PasswordField.jsx

```diff
@@ -83,6 +94,7 @@ class PasswordField extends PureComponent {
 
 PasswordField.defaultProps = {
   autoComplete: false,
+  describedBy: null,
```

Comment:
```suggestion
  describedBy: '',
```
Vu que c'est une string.